src/state/swap/hooks.tsx

```diff
@@ -104,6 +107,12 @@ export function useDerivedSwapInfo(state: SwapState, chainId: ChainId | undefine
 
   const inputCurrency = useCurrency(inputCurrencyId, chainId)
   const outputCurrency = useCurrency(outputCurrencyId, chainId)
+
+  const { inputTax, outputTax } = useSwapTaxes(
+    inputCurrency?.isToken ? inputCurrency.address : undefined,
```

Comment:
can't wait till we have to support a chain that's native has a FOT

Reply:
💀💸